### PR TITLE
787 fix catalog release script

### LINKED ISSUE #787: [BUG] Fix Catalog Dockerfile and release script
**Describe the bug**
The Catalog release script was incorrectly using the Contributions BB Dockerfile. Also, the Dockerfile needs to be updated to fix a path. 

**Expected behavior**
Catalog running using the updated Docker image both locally and in the DEV instance.

**Additional context**

### PR DESCRIPTION
**Pull request by sandeep-ps**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

This PR fixes the current Catalog release script, Dockerfile, and also adds Docker instructions to the Catalog README. 

**Fixes #787**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] My change requires updating the documentation.
- [x] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [x] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Review by ywkim312 — DISMISSED** [review]

File: contributions/catalog/README.md [hunk 1]
```diff
@@ -56,3 +56,27 @@ set FLASK_ENV=development
...
+For a list of available configurations, please see `controllers/config.py`.
+
+
+### AWS ECR Instructions
+
+Make sure that a repository called rokwire/contributions-catalog exists in ECR. Then, create the Docker image for Contributions Catalog and push to AWS ECR using:
+
+```
+cd rokwire-building-blocks-api
+$(aws ecr get-login --no-include-email --region us-east-2)
```
Comment (minump) [inline contributions/catalog/README.md:78]: aws ecr get-login is deprecated. Might need to use aws ecr-get-login-password instead.
I used the below command to both login and pass the password for docker login
```
aws ecr get-login-password --region us-east-2 | docker login -u AWS --password-stdin 779619664536.dkr.ecr.us-east-2.amazonaws.com
```

**Review by minump — DISMISSED** [review]
@sandeep-ps Looks good. Just a change in the aws ecr get-login command.

File: contributions/catalog/README.md (hunk 1 above)
Comment (sandeep-ps) [reply contributions/catalog/README.md:78]: Thanks. I will try this command with others in this section and update the README accordingly.

**Comment by sandeep-ps** [comment]
@minump, I have updated this PR. Please check when you get a chance.

**Review by minump — DISMISSED** [review]

**Review by minump — APPROVED** [review]
approved

**Pull request merged**